src/RobotInterface/YarpImplementation/include/BipedalLocomotion/RobotInterface/YarpSensorBridgeImpl.h

```diff
@@ -0,0 +1,1695 @@
+/**
+ * @file YarpSensorBridgeImpl.h
+ * @authors Prashanth Ramadoss
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_ROBOT_INTERFACE_YARP_SENSOR_BRIDGE_IMPL_H
+#define BIPEDAL_LOCOMOTION_ROBOT_INTERFACE_YARP_SENSOR_BRIDGE_IMPL_H
+
+#include <BipedalLocomotion/RobotInterface/YarpSensorBridge.h>
+#include <BipedalLocomotion/GenericContainer/Vector.h>
+
+// YARP os
+#include <yarp/os/Time.h>
+
+// YARP sig
+#include <yarp/sig/Vector.h>
+#include <yarp/sig/Image.h>
+
+// YARP Sensor Interfaces
+#include <yarp/dev/IAnalogSensor.h>
+#include <yarp/dev/IGenericSensor.h>
+#include <yarp/dev/MultipleAnalogSensorsInterfaces.h>
+
+// YARP Camera Interfaces
+#include <yarp/dev/FrameGrabberInterfaces.h>
+#include <yarp/dev/IRGBDSensor.h>
+
+// YARP Control Board Interfaces
+#include <yarp/dev/IEncodersTimed.h>
+#include <yarp/dev/IAxisInfo.h>
+
+// std
+#include <cmath>
+#include <set>
+#include <algorithm>
+
+using namespace BipedalLocomotion::RobotInterface;
+using namespace BipedalLocomotion::GenericContainer;
+
+namespace BipedalLocomotion
+{
+namespace RobotInterface
+{
+
+struct YarpSensorBridge::Impl
+{
+    using StampedYARPVector = std::pair<yarp::sig::Vector, double>;
+    using StampedYARPImage = std::pair<yarp::sig::Image, double>;
+
+    /**
+     * Struct holding remapped remote control board interfaces
+     */
+    struct
+    {
+        yarp::dev::IEncodersTimed* encoders;
+        yarp::dev::IAxisInfo* axis;
+    } controlBoardRemapperInterfaces;
```

Comment:
This is the c style for defining a struct you can write  
```suggestion
    struct ControlBoardRemapperInterfaces
    {
        yarp::dev::IEncodersTimed* encoders;
        yarp::dev::IAxisInfo* axis;
    };
```

Reply:
Fixed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/106/commits/a50d061e44dd2acc504bcd578df9f24901a0073f